### ISSUE TITLE
not working with Anki 2.1.47

### ISSUE THREAD

**Issue by sonicscrewdriverau** [Arthur-Milchior/anki-copy-note#35]
I have used this add-on repeatedly for quite a while.  It has been fantastic.  It now doesn't function.   I disabled all other add-ons then attempted to run Copy-Note but it still did not work.  Seems to be very similar error to #34.  See error details below:

Error 
An error occurred. Please start Anki while holding down the shift key, which will temporarily disable the add-ons you have installed. 
If the issue only occurs when add-ons are enabled, please use the Tools>Add-ons menu item to disable some add-ons and restart Anki, repeat until you discover the add-on that is causing the problem. 
When you've discovered the add-on that is causing the problem, please report the issue on the add-ons section of our support site. 
Debug info:
Anki 2.1.47 (dc156f34) Python 3.8.6 Qt 5.14.2 PyQt 5.14.2
Platform: Windows 10
Flags: frz=True ao=True sv=2
Add-ons, last update check: 2021-09-07 05:29:45

Caught exception:
Traceback (most recent call last):
  File "C:\Users\...\AppData\Roaming\Anki2\addons21\1566928056\copyNote.py", line 67, in <lambda>
    a.triggered.connect(lambda: copyNotes(browser.selectedNotes()))
  File "C:\Users\...\AppData\Roaming\Anki2\addons21\1566928056\copyNote.py", line 54, in copyNotes
    copyNote(nid)
  File "C:\Users\...\AppData\Roaming\Anki2\addons21\1566928056\copyNote.py", line 87, in copyNote
    copyCard(card, note)
  File "C:\Users\...\AppData\Roaming\Anki2\addons21\1566928056\copyNote.py", line 110, in copyCard
    card.flush()
  File "anki\cards.py", line 121, in flush
Exception: card.flush() expects an existing card

**Comment by ijgnd** [comment]
@sonicscrewdriverau : Maybe relevant for you, else ignore it:

I often use arthur's add-on and missed it for .45+. So I made some changes in the code that seem to work after a quick test but that still could have some problems. If you have some time left over and a secondary account that you can afford to mess up I could need some help with testing my changes at https://github.com/ijgnd/anki-copy-note-1/issues/1#issuecomment-917493893 

basically I changed two files in arthur's add-on: https://github.com/ijgnd/anki-copy-note-1/blob/master/new_note_id.py and https://github.com/ijgnd/anki-copy-note-1/blob/master/copyNote.py  So you could copy them over into the addons21 folder and restart anki and then try out the changes. As I said only test with a throw-away collection and account. do not use it on your regular collection unti you've done enough testing.
